spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -19,27 +19,33 @@ include:
 definitions:
 
  - ObsGPSTime:
-    short_desc: Millisecond-accurate GPS time
+    short_desc: Nanosecond-accurate receiver clock time
     desc: |
-      A wire-appropriate GPS time, defined as the number of
-      milliseconds since beginning of the week on the Saturday/Sunday
+      A wire-appropriate receiver clock time, defined as the time
+      since the beginning of the week on the Saturday/Sunday
       transition.
     fields:
         - tow:
             type: u32
             units: ms
             desc: Milliseconds since start of GPS week
+        - ns:
```

Comment:
Why not to use Q32 data format here? In a manner, similar to examples below, where Q8 is used? Then the accuracy becomes 1/2^32 ms.